src/Table/TableRow.scss

```
@@ -0,0 +1,15 @@
+@import '../../scss/theme.scss';
+
+.TableRow {
+  &:hover {
+    background-color: $ux-gray-100;
+  }
+
+  &--clickable {
+    cursor: pointer;
+  }
+
+  &--selected {
+    background-color: $ux-blue-100 !important;
```

Just checking -- are these `!important` flags here because we want to stop people from overriding or because it's hard to pin down the specificity? In general `!important` scares me as it really limits a consumer from being able to style things. I imagine we could do:
```suggestion
    background-color: $ux-blue-100; 
    
    &:hover {
      background-color: $ux-blue-100;
    }
```
To override the specificity of the above `&:hover` style and should prefer that to using `!important` purely as a way to fight specificity issue. (Note -- this applies to other uses of `!important` in this PR.

---

We decided to give a different background color to the `TableRow--selected` when hovered, so that get rids of `!important` 

https://user-images.githubusercontent.com/37383785/116468765-1d603500-a826-11eb-8b3f-46e492b5a96b.mov